astropy/table/pprint.py

```diff
@@ -213,15 +213,13 @@ def _get_pprint_size(max_lines=None, max_width=None):
             max_lines = lines
         elif max_lines < 0:
             max_lines = sys.maxsize
-        if max_lines < 8:
-            max_lines = 8
+        max_lines = max(max_lines, 8)
 
         if max_width is None:
             max_width = width
         elif max_width < 0:
             max_width = sys.maxsize
```

Comment:
What about this?

Reply:
How should it be refactored ? I don't see it.

Reply:
Ah, right. I guess the comparisons are different enough we can't turn it into a one-liner. Never mind. Sorry for the noise.